### ISSUE TITLE
Removal of a space between bolded preceeding text and hyperlinks succeeding text

### ISSUE THREAD

**Issue by DanielSchuman** [benbalter/word-to-markdown#87]
### Removal of a space in certain circumstances

In a google document, when the preceeding text is bolded and the succeeding text is hyperlinked, an intervening space will be omitted in the output.

For example, this:

![image](https://user-images.githubusercontent.com/2955984/47271436-62190000-d547-11e8-96c9-98c4a618910a.png)

Becomes this:

![image](https://user-images.githubusercontent.com/2955984/47271443-78bf5700-d547-11e8-8cd4-de381ab97877.png)

In other words, this:

**Pelosi said she might be a short-term (transitional) speaker** [should the Dems](http://www.latimes.com/politics/la-na-pol-nancy-pelosi-speaker-20181019-story.html) 

Becomes this:

**Pelosi said she might be a short-term (transitional) speaker**[should the Dems](http://www.latimes.com/politics/la-na-pol-nancy-pelosi-speaker-20181019-story.html) 

Notice the removal of the space between the words 'speaker' and 'should'
